com.yubico.YubiOath.appdata.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
As usually, this should be contributed upstream.

Reply:
Do you mean that we should ask upstream to distribute this file?

Reply:
Yes. It's a freedesktop standard & not only a Flathub thing. Other packages could benefit from a nice appdata file.

Reply:
I sent PR to upstream with fixes suggested below https://github.com/Yubico/yubioath-desktop/pull/524